src/shared/components/schoolCard/schoolCard.css

```diff
@@ -2,10 +2,10 @@
   display: flex;
   border-radius: 4px;
   align-items: center;
-  margin: 20px 10px;
+  margin: 25px 10px;
```

Comment:
Exnay all the changes to the school card styles.

All of the text on every card is now a link and it barely fits any of the cards.